.cicd/generate-pipeline.sh

```diff
@@ -732,12 +697,12 @@ cat <<EOF
 
   - label: ":beer: Brew Updater"
     command:
-      - "buildkite-agent artifact download eosio.rb . --step ':darwin: macOS 10.14 - Package Builder'"
+      - "buildkite-agent artifact download eosio.rb . --step ':darwin: macOS 10.15 - Package Builder'"
       - "buildkite-agent artifact upload eosio.rb"
     agents:
       queue: "automation-basic-builder-fleet"
     timeout: "${TIMEOUT:-5}"
-    skip: ${SKIP_PACKAGE_BUILDER}${SKIP_MAC}${SKIP_MACOS_10_14}
+    skip: ${SKIP_PACKAGE_BUILDER}${SKIP_MAC}
```

Comment:
This line...
```yaml
skip: ${SKIP_PACKAGE_BUILDER}${SKIP_MAC}${SKIP_MACOS_10_14}
```
...should become...
```yaml
skip: ${SKIP_PACKAGE_BUILDER}${SKIP_MAC}${SKIP_MACOS_10_15}
```
...as this step now depends upon Catalina. If Catalina is skipped (by setting `SKIP_MACOS_10_15`), this step will fail.

Perhaps that is the wrong thing to do and this step should be pulling `brew` packages from macOS 11, but that is a larger discussion. For the purposes of this story, setting it to Catalina is equivalent to how it behaved before the upgrade.

Reply:
Now it is skip: ${SKIP_PACKAGE_BUILDER}${SKIP_MAC}${SKIP_MACOS_10_15}